monitor_version_scripts/web_scrap_files/dotnet_versions.py

```diff
@@ -0,0 +1,120 @@
+from bs4 import BeautifulSoup
```

Comment:
Can we use this API to get latest versions instead of crawling the webpage?
https://endoflife.date/docs/api

Reply:
Yes, updated node_versions, php_versions , python_versions to take latest versions from the api